CHANGELOG.md

```
@@ -1,8 +1,16 @@
-## [5.0.0]
+
+## [5.0.0] - 31/08/2026
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)